rustls/src/server/server_conn.rs

```diff
@@ -564,9 +564,6 @@ mod connection {
         /// Make a new ServerConnection.  `config` controls how
         /// we behave in the TLS protocol.
         pub fn new(config: Arc<ServerConfig>) -> Result<Self, Error> {
-            let mut common = CommonState::new(Side::Server);
```

Comment:
Thanks -- good spot! There is also another instance of this in `UnbufferedServerConnection::new` -- care to eliminate that one too?